### PR TITLE
Assignment 2b

### PR DESCRIPTION
**Pull request by patin9**

### CODE REVIEW

**Comment by patin9** [comment]
Sorry, I committed the files yesterday but forgot to ask for the pull request.

File: Assignment2/patin9-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		"assig": "http://assignment2.com/2b#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"name": "assig:name",
+		"homeTown": "assig:homeTown",
+		"age": "assig:age",
+		"almaMater": "assig:almaMater",
+		"parentsName": "assig:parentsName",
+        "subjects": "assig:subjects"
+	},
+	"name": "Eva Tagarro",
```
Comment (paoespinozarias) [inline Assignment2/patin9-fromJSONtoJSON-LD.jsonld:12]: Add the. "@id" and "@type"

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/patin9-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        {
+          "@id": "assig:Sensor029"
+        },
+        {
+          "@id": "assig:Computer101"
+        }
+      ]
+    }
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/patin9-fromRDFtoJSON-LD.jsonld:46]: This file is ok.

File: Assignment2/patin9-fromJSONtoJSON-LD.jsonld (hunk 1 above)
Comment (paoespinozarias) [reply Assignment2/patin9-fromJSONtoJSON-LD.jsonld:14]: Be careful when reusing prefixes. You don't have any assignment prefix on your context definitions. The correct definition is "@type": "assign:student"

**Review by paoespinozarias — COMMENTED** [review]
Assignment 2b is ok. Now @ocorcho will review Assignment 3

**Pull request merged**